tests/test_join.py

```diff
@@ -105,3 +105,17 @@ def test_join_random(seed, lt):
     assert djoined.shape == (ndata, 2)
     assert djoined.names == ("KEY", "VAL")
     assert djoined.topython() == [main, res]
+
+
+
+def test_join_update():
+    d0 = dt.Frame([[1, 2, 3, 2, 3, 1, 3, 2, 2, 1], range(10)], names=("A", "B"))
+    d1 = d0[:, mean(f.B), f.A]
+    d1.key = "A"
+    d0[:, "AA", join(d1)] = g.V0
```

Comment:
update made is by reference, correct?

Reply:
The LHS table (`d0`) is updated in-place. The RHS needs to be computed in order to match the shape of the LHS.